app/assets/stylesheets/categories.scss

```diff
@@ -0,0 +1,109 @@
+$font_logo: 'Caveat', cursive;
+$font_text: 'Outfit', sans-serif;
+$blue_color: #3778c2;
+$green_color: #5fb523;
+$grey_color: #434b54;
+
+.categories_section {
+  height: 75vh;
+  display: flex;
+  flex-direction: column;
+}
+
+.new_category_div a {
+  color: white;
+  text-decoration: none;
+  font-family: $font_text;
+  border: 3px solid white;
+  border-radius: 5px;
+  width: 90%;
+  height: 70%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-weight: bold;
+  background-color: $green_color;
+  transition: all 0.4s;
+}
+
+.categories_small_div .total {
+  margin-right: 1rem;
+}
+
+.categories_major_div {
+  display: flex;
+  flex-direction: column;
+  max-height: 100%;
+  overflow-y: auto;
+
+  a {
+    text-decoration: none;
+    font-family: $font_text;
+    color: rgb(177, 177, 177);
+    font-weight: bold;
+    transition: all 0.4s;
+
+    .total {
+      color: black;
+    }
+  }
+
+  a:hover {
+    transform: scale(1.02);
+  }
+}
```

Comment:
- [ ] Please let us work on the following in this section:

![category](https://user-images.githubusercontent.com/45002654/144164069-7fafc341-af03-4a74-b44c-df6c7cafa6bc.png)

1. The image/icon
Currently, we can only add dummy text to represent an image. Part of the test is to display an actual image icon for categories. Please follow this [guide](https://levelup.gitconnected.com/rails-image-upload-101-f9bf245e389b) to help with implementing a simple image upload.
When you finally deploy your app, you might see images broken, you can use a free cloud hosting service to help with this. Cloudinary is a good one to consider. Follow this [guide](https://medium.com/nerd-for-tech/getting-started-with-cloudinary-in-ruby-on-rails-6-925888032395) for this.

2. Alignment
Text alignment in the original design is with the image. That is, the category name is aligned left, next to the image. It would be nice if you refactored this to help be almost at par with the original design. See the screenshot below and compare it with the one above to help make comparisons.
![category](https://user-images.githubusercontent.com/45002654/144164729-8775df65-7f19-4e18-9f3a-f80d4ba9325f.png)

Reply:
Well done here!